engine-standalone-nep141-legacy/src/legacy_connector.rs

```diff
@@ -614,7 +614,7 @@ impl<I: IO + Copy> EthConnectorContract<I> {
             )?,
         }
         self.save_ft_contract();
-        self.io.return_output("\"0\"".as_bytes());
+
```

Comment:
@joshuajbouw said about another function. We figured out this. But what is confused me a bit that we just return a  "0" here. But the standard says:
> Returns a value or a promise which resolves with a value. The value is the number of unused tokens in string form. For instance, if `amount` is 10 but only 9 are needed, it will return "1".

cc @joshuajbouw @birchmd

Reply:
> How did you get it?

I thought the difference of deposit amount was from different gas cost from serialization in the error related to those tests, but it turns out I was wrong.